resource-aggregate/pb/events.proto

```diff
@@ -205,3 +205,28 @@ message DeviceMetadataSnapshotTaken {
     // Open telemetry data propagated to asynchronous events
    // map<string,string> open_telemetry_carrier = 100;
 }
+
+
+message ServicesHeartbeat {
+    message Heartbeat {
+        string service_id = 1; // generated unique id during start the service
+        int64 heartbeat_valid_until = 2; // unix timestamp in nanoseconds (https://golang.org/pkg/time/#Time.UnixNano) when service heartbeat is considered as expired.
+    }
+    repeated Heartbeat online = 1; // services which are online
+    repeated Heartbeat offline = 2; // services which are offline
```

Comment:
heartbeat can be expired or valid, not online or offline

Reply:
services which heartbeat is already expired
services which heartbeat is still valid

--> doc